### DISCUSSION TITLE
Wrong forward of http://muse-sequencer.org

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1132] [Team Posts]
Just a heads up that I forwarded this problem to Andrew (he might even be reading this message now :) ) as I don't think I can fix it from the access I have to the server (at least I don't know how).

**Comment by kybos** [comment]
The http page is still wrong. https is working for the front page, but there the wiki is broken, all the images are lost, can't be displayed.
Where is the web (and wiki) physically? How is the access to it organized?
Would it be an option to move the wiki to github for more transparency? Some revamping would do it good anyway...

**Comment by kybos** [comment]
It looks like the complete web could be hosted on github:
https://help.github.com/en/github/working-with-github-pages
The html code could then be integrated in the normal development workflow.

**Comment by spamatica** [comment]
The web is hosted by Andrew who used to be very active in the development,
now he just hosts the page. It is somewhere in Russia I don't know exactly
where.
It seems he has very little time at the moment though, I have sent mail to
him about the wrong forward but he has not replied yet.

About hosting on github, very recently I noticed that this is an option
too. Would be good to hear from Andrew first but maybe this would be the
best long term solution?
And yes, it would be good to give it all a face lift.

Den tis 10 dec. 2019 kl 06:51 skrev kybos <notifications@github.com>:

**Comment by kybos** [comment]
No news here? It means that we have no control over the domain? 
The domain seems to be registered in Sweden. I assume somebody has to pay for it? Are there any donations that come up for that? Just curious - don't worry, I don't want a share, don't care about money, that's certainly not my motivation ;-).
I hope you don't mind if I experiment with the GitHub Pages a little...

**Comment by spamatica** [comment]
Sorry for not replying, the world turned and I did not turn with it.
Sprinting!

And in slightly less confusing, less poetic words...  I own the domain and
should have all that is needed to move it. I was just hoping we would hear
from Andrew but he seems very occupied.
Maybe we should start preparing to move to github? Look like it could be a
good solution, a stable provider and easy access for all developers.

As for designing a new web. I think about this everytime I go to the
homepage.
Did you have a design in mind? I've grown to like the scrolling frontpages
like this one that came up a while ago: https://surge-synthesizer.github.io/.
Though I think the images there are too big.

We should move away from the logo used on the current page too, no one
knows what font was used and the enlarged version there looks awful.


Den fre 13 dec. 2019 kl 07:43 skrev kybos <notifications@github.com>:

**Comment by kybos** [comment]
No problem, I am quite busy myself, it's always that time before Christmas...
I looked into github Pages and they seem pretty usable for that purpose (I don't think we need server-side things like php etc.).
No, I don't have any concrete design in mind, you are free to put something nice up ;-). What disturbed me was the obsolete content in the wiki pages in the first place. But a new design is of course a good idea too.
